### PR TITLE
A0-1668 Make runtime cache configurable

### PR DESCRIPTION
**Pull request by krzysztofziobro**
# Description
Making runtime cache configurable (so we can increase it for RPC nodes)

## Type of change
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: docker/docker_entrypoint.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ TELEMETRY_URL=${TELEMETRY_URL:-'wss://telemetry.polkadot.io/submit/'}
 TELEMETRY_VERBOSITY_LVL=${TELEMETRY_VERBOSITY_LVL:-'0'}
 UNIT_CREATION_DELAY=${UNIT_CREATION_DELAY:-300}
 DB_CACHE=${DB_CACHE:-1024}
+RUNTIME_CACHE_SIZE=${RUNTIME_CACHE_SIZE:-2}
```
[the remark this answers is not in the document]
Comment (krzysztofziobro) [reply docker/docker_entrypoint.sh:34]: If we only want to set it for archivists, then it's probably easier to keep it at default, and increase it only for specific nodes.
But it's probably better to make a "mistake" of having runtime cache "too large" than too small?

[the remark this answers is not in the document]
Comment (krzysztofziobro) [reply docker/docker_entrypoint.sh:34]: Unfortunately, I do not know how much memory that is
From resources used by process it looks like each cached runtime takes up something like ~20-80 MB (does it seem right?)

[the remark this answers is not in the document]
Comment (krzysztofziobro) [reply docker/docker_entrypoint.sh:34]: So combined cost would be several hundred MB of RAM

[the remark this answers is not in the document]
Comment (DamianStraszak) [reply docker/docker_entrypoint.sh:34]: Oh, this seems like a lot. So maybe better to set default to 2 actually.

**Review by fixxxedpoint — APPROVED** [review]

**Pull request merged**